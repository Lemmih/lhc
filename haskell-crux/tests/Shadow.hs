module Shadow where

x x = \x -> \x -> \x -> x
