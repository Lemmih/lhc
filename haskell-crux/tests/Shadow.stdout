Shadow.x : ∀ v1 v3 v5 v8. v1 →  v3 →  v5 →  v8 →  v8
Shadow.x =
  λ @.v1^1:* @.v3^2:* @.v5^3:* @.v8^4:* Shadow.x^5:v1 Shadow.x^6:v3 Shadow.x^7:v5 Shadow.x^8:v8 → 
  Shadow.x^8
